src/Configuration/src/CloudFoundry/ServiceBindings/PostProcessors/PostgreSqlCloudFoundryPostProcessor.cs

```diff
@@ -25,9 +25,11 @@ public override void PostProcessConfiguration(PostProcessorConfigurationProvider
             mapper.MapFromTo("credentials:name", "database");
             mapper.MapFromTo("credentials:username", "username");
             mapper.MapFromTo("credentials:password", "password");
+            // ReSharper disable StringLiteralTypo
```

Comment:
Any reason not to change the casing to like sslCert instead of suppressing?

Reply:
You're right, these are read from `IConfiguration`, where keys are case-insensitive. So there's no need to assume the original casing from the JSON in `VCAP_SERVICES`. I've removed the suppression and changed the keys to camel case.